commands/system.go

```diff
@@ -95,3 +105,18 @@ func clearBusyCmdF(c client.Client, cmd *cobra.Command, _ []string) error {
 	printer.PrintT("Busy state cleared", map[string]string{"status": "ok"})
 	return nil
 }
+
+func systemVersionCmdF(c client.Client, cmd *cobra.Command, _ []string) error {
+	printer.SetSingle(true)
+	// server version information comes with all responses. We can't
+	// use the initial "withClient" connection information as local
+	// mode doesn't need to log in, so we use an endpoint that will
+	// always return a valid response
+	_, resp := c.GetAllTeams("", 0, 0)
```

Comment:
Why not uses for example the `/system/ping` endpoint instead, that doesn't hit the database and probably gets you the same data?

Reply:
Actually, having a server-status command would be great too :)

Reply:
Completely missed the fact that the ping endpoint didn't require of authentication, thanks for pointing it out! Will push a fix and a server PR to add the ping endpoint to the local API